### ISSUE TITLE
Tribute Page beta challenge id="img-div" not recognizing child id's "image" and "img-caption"

### ISSUE THREAD

**Issue by DarrenfJ** [freeCodeCamp/testable-projects-fcc#34]
for @no-stack-dub-sack 

#### Issue Description
original ticket here: https://github.com/freeCodeCamp/freeCodeCamp/issues/13065#

pen here:  http://codepen.io/DarrenfJ/pen/jAxadd

Fails last 3 tests. Seems to not recognize that the id's img-caption and image are inside the div with id img-div. This was confirmed when I commented out 2 lines that linked to a youtube vid between the div img-div and the other 2

#### Browser Information
see original ticket, it captured that information

* Browser Name, Version: 
* Operating System: 
* Mobile, Desktop, or Tablet: 

#### Your Code
commented out the following and it passed all 9 tests (line 17 and 18 and corresponding closing div line 21
```
<!-- <div class="text-center">
            <a href="https://youtu.be/lLvsnkQtZLI" target="_blank"> --> 
```
between `id="img-div"` and `id="image"` and `id="img-caption"`

**Comment by no-stack-dub-sack** [comment]
@DarrenfJ you can go ahead and re-fix your pen if you'd like - I've forked your project [here](http://codepen.io/no_stack_dub_sack/pen/oBdmZO) to preserve the issue. Thanks for reporting this! I'll look into this and should be able to fix.

**Comment by no-stack-dub-sack** [comment]
@DarrenfJ Thanks again for pointing this out - I've fixed the tests so now [the broken one](http://codepen.io/no_stack_dub_sack/pen/oBdmZO?editors=1111) that I forked to preserve still works. The issue was that the test was previously assuming `id="image"` & `id="img-caption"` would be a direct descendents of `id="img-div"`, so it was checking for their direct parent, expecting it to be "img-div" when really, in this case, it was the div with class "text-center". 

So I refactored the tests to use jQuery's `find()` method which can find a child element nested at an arbitrary depth. 

The last test was failing for a different reason - because it was trying to grab the id of the immediate parent element, which in this case, the `<a>` element did not have an id. So this is fixed as well. 

However, I did want to point out that, even though this did not affect the tests, you do have an unclosed tag in your example. The anchor tag surrounding the image was not closed, I closed it here:
![image](https://cloud.githubusercontent.com/assets/18563015/22614244/ae1f7bdc-ea4d-11e6-99de-42e49d038618.png)

Anyway, no big deal! Thanks again for helping to improve this!
